test/integration/custom_entity_test.go

```diff
@@ -0,0 +1,285 @@
+//go:build integration_tests
+
+package integration
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/kong/go-kong/kong"
+	"github.com/kong/go-kong/kong/custom"
+	"github.com/kong/kubernetes-testing-framework/pkg/clusters"
+	"github.com/kong/kubernetes-testing-framework/pkg/utils/kubernetes/generators"
+	"github.com/samber/lo"
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/kong/kubernetes-ingress-controller/v3/internal/adminapi"
+	dpconf "github.com/kong/kubernetes-ingress-controller/v3/internal/dataplane/config"
+	kongv1 "github.com/kong/kubernetes-ingress-controller/v3/pkg/apis/configuration/v1"
+	kongv1alpha1 "github.com/kong/kubernetes-ingress-controller/v3/pkg/apis/configuration/v1alpha1"
+	"github.com/kong/kubernetes-ingress-controller/v3/pkg/clientset"
+	"github.com/kong/kubernetes-ingress-controller/v3/test"
+	"github.com/kong/kubernetes-ingress-controller/v3/test/consts"
+	"github.com/kong/kubernetes-ingress-controller/v3/test/internal/helpers"
+)
+
+func TestCustomEntity(t *testing.T) {
```

Comment:
Please consider converting it to an isolated integration test (like e.g. https://github.com/Kong/kubernetes-ingress-controller/blob/main/test/integration/isolated/udproute_test.go#L33). I believe we agreed that we'd no longer add new regular integration tests in favor of the isolated ones as they allow configuring the environment granularly (i.e. you could enable the feature gate only in the test).

Reply:
Moved to isolated tests.